### PR TITLE
Hide Payment Request container until mounting Element

### PR DESCRIPTION
**Pull request by dechov**
Fixes https://github.com/woocommerce/woocommerce-gateway-stripe/issues/855

#### Changes proposed in this Pull Request:

Hide Payment Request container until the Element is actually being mounted there. This affects the loading state but primarily it improves cases where no button would show.

Testing in Firefox:

`master` | this branch
-- | --
 | 

I also verified that the appearance is unchanged when the button does show (testing with jurassic.ninja).

-------------------
- [x] Make sure your changes respect [WordPress' coding standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
- [x] Did you make changes, or create a **new .js file**? If **Gruntfile.js** exists in the repo, make sure to run `grunt`.

### CODE REVIEW

File: assets/js/stripe-payment-request.js [hunk 1]
```diff
@@ -406,12 +406,9 @@ jQuery( function( $ ) {
 					}
 
 					if ( $( '#wc-stripe-payment-request-button' ).length ) {
+						$( '#wc-stripe-payment-request-wrapper, #wc-stripe-payment-request-button-separator' ).show();
```
Comment (DanReyLop) [inline assets/js/stripe-payment-request.js]: Why did you move the `show()` call to before the `mount()` call?

Comment (dechov) [reply assets/js/stripe-payment-request.js]: Not sure if it makes a difference but I'd felt there was a slim chance that the mounting of the button could suffer from the container being hidden (if it reads any layout measurements, for instance), and it made sense to me that the separator would also just be shown earlier as well. Didn't actually test it the other way, but that was why – do you feel differently / prefer to keep one or both elements hidden until after mounting?

Comment (DanReyLop) [reply assets/js/stripe-payment-request.js]: I was just curious if this change was intentional, since before this `mount()` was before `show()` and seemed to be working well. Your explanation makes sense, and worst-case-scenario, it just makes no difference :)
